packages/plugin-fastlane/src/index.ts

```diff
@@ -1,9 +1,73 @@
-const ios = () => {
-  //
+import fs from "fs-extra";
+import ejs from "ejs";
+
+import { Config, path, logger } from "@brandingbrand/kernel-core";
+import { KernelPluginFastlane } from "./types";
+
+interface PathConfig {
+  platform: "ios" | "android";
+  path: typeof path.ios | typeof path.android;
+}
+
+const initFastFile = async (
+  pathConfig: PathConfig,
+  config: Config & KernelPluginFastlane
+) => {
+  const templateDir = path.resolve(
+    path.project.pluginPath("plugin-fastlane"),
+    "src",
+    "fastlane.template",
+    pathConfig.platform,
+    "fastlane"
+  );
```

Comment:
this does not seem correct.....remember this is going to be bundled i.e. a single js file inside the dist folder - the template folder will be in the same directory